src/client/pages/CubePlaytestPage.tsx

```diff
@@ -56,7 +56,7 @@ const CubePlaytestPage: React.FC<CubePlaytestPageProps> = ({ cube, decks, decksL
 
   return (
     <MainLayout loginCallback={loginCallback}>
-      <CubeLayout cube={cube} activeLink="playtest" hasControls={user !== null && cube.owner.id === user.id}>
```

Comment:
I tested with this original line and the RenderToRoot change, and in that case there was no bug because the User context was accurately a User object or null, no undefined. Either way, using less strict check again mainly for consistency